### DISCUSSION TITLE
Back with a vengeance! Amazing framework, super extensible! THANK YOU!

### DISCUSSION THREAD

**Discussion by PizzaConsole** [PiranhaCMS/piranha.core#1681] [Show and tell]
Hi all, I started looking at Piranha Core right before the Vuejs manager was implemented like in 2019 (I forget the version). I did not end up implementing the CMS in production and I looked around at several other things. I was pretty green to .NET and Software development in general. since then I have really beefed up my knowledge and I work as a full-time Software Development Engineer using .NET and various other technologies.

In the past two weeks I came back to look at PiranhaCMS because I have some old WordPress Sites that I **really** wanted OFF WordPress. In those two weeks I have **fallen in love** with PiranhaCMS! I have in production about 12 domains and 5 app instances all running off a singular implementation of PiranhaCMS. 

I have also implemented my own extensions for [hCaptcha](https://www.hcaptcha.com/), [Dynamic Forms](https://formbuilder.online/), [Plausible Analytics](https://plausible.io/), and [Email API](https://www.mailgun.com/email-api/). I have also figured out how to run [Blazor-ServerSide](https://docs.microsoft.com/en-us/aspnet/core/blazor/hosting-models?view=aspnetcore-5.0#blazor-server) pages in my Manager portal as well. 

### All in all I just wanted to say THANK YOU for this amazing framework. I expect to be contributing in anyway possible as I continue my development.

**Comment by pt4r** [comment]
Hey @PoisnFang i know its been a minute since your post but would be so kind to share some of the implementations or any tutorials? It would be wonderful especially for the Dynamic Forms.
